README.md

```diff
@@ -37,19 +37,21 @@ Welcome to our community! We welcome contributors from many skillsets. Here's ho
-|---|---|---|---|---|
-| Website for EDGI 100 Days Report at [100days. envirodatagov.org](https://100days.envirodatagov.org/) [**github.com/../100days**](https://github.com/edgi-govdata-archiving/100days) | Set of tools, workflows, and documentation for grassroots, event-driven, archiving | Distributed model for holding copies of archived and preserved data [**github.com/ datatogether**](https://github.com/datatogether) | Interface and backend for reviewing different versions of web pages [**github.com/../web-monitoring**](https://github.com/edgi-govdata-archiving/web-monitoring) | Project management and design support for EDGI's website at [envirodatagov.org](https://envirodatagov.org/) [**github.com/../edgi-website**](https://github.com/edgi-govdata-archiving/edgi-website) |
-
-There are additional [`community`](https://github.com/edgi-govdata-archiving/overview/labels/community) and [`infrastructure`](https://github.com/edgi-govdata-archiving/overview/labels/infrastructure) projects that support internal EDGI and outreach processes:
-
-| Tool Name | Description |
-|-----------|-------------|
-| [**EDGI Hubot**](https://github.com/edgi-govdata-archiving/edgi-hubot) | Chat bot for EDGI Slack built on the Hubot framework  |
-| [**EDGI Scripts**](https://github.com/edgi-govdata-archiving/edgi-scripts) | Code scripts for running and maintaining our digital infrastructure |
-| [**Video Call Landing Page**](https://github.com/edgi-govdata-archiving/video-call-landing-page) | Landing page app with important info that participants can be sent through prior to joining a video call  <br />[http://edgi-video-call-landing-page.herokuapp.com/](http://edgi-video-call-landing-page.herokuapp.com/) |
+Here are some projects we're building and maintaining right now.
+
+**Want to get involved?** Check out the emoji column to see all the different types of contribution we need for these projects!
+
+| Project (Click through to repo) | Description | Contribution type most needed ([emoji key from All Contributors](https://github.com/kentcdodds/all-contributors#emoji-key))|
+| --- | --- | --- |
+| [Web Monitoring](https://github.com/edgi-govdata-archiving/web-monitoring) | Tools around monitoring changes to government websites | 📖  🐛 💻
+| [Walk](https://github.com/qri-io/walk) (Web Monitoring and Archiving) | A system for scraping a BIG list of URLs and processing the results for monitoring and archiving | ✅ ⚠️ 📖 💻 🐛 👀
+| [EIS Search Tool](https://github.com/edgi-govdata-archiving/eis-search)  | Making federal environmental impact statements easier to search | 🤔 🐛 💻 ⚠️ 📖 🎨 📓
+| [Data Together](https://github.com/datatogether/datatogether) | Developing a distributed model for holding copies of archived and preserved data by reading, talking, and prototyping together | 📝 🎨 💡 📋 🤔 📢 🚇
+|[100 Days](https://github.com/edgi-govdata-archiving/100days) | Website for EDGI 100 Days Report at [100days. envirodatagov.org](https://100days.envirodatagov.org/) | 🐛 🚇 📖 📢
```

Comment:
This project is mostly just in maintenance mode, right? Should it just need bug fixing?

Reply:
Good question for I think @lourdesvera